src/main/commandServer/httpCommandServer.ts

```diff
@@ -142,6 +151,37 @@ export class HttpCommandServer {
     return Promise.resolve();
   }
 
+  protected getPathsForVersion(version: string, returnedPaths: string[]): string[] {
+    const paths = this.dispatchTable[version];
+
+    returnedPaths.push(`GET /${ version }`);
+    for (const method in paths) {
+      for (const path in paths[method]) {
+        returnedPaths.push(`${ method } ${ ['', version, path].join('/') }`);
+      }
+    }
+
+    return returnedPaths;
```

Comment:
Should we do the sort here? It would be nice to have stable output.

Reply:
Sure, but that's the wrong place to do the sort, because one of the calling paths writes in a hardwired `GET /`. Turns out that's always going to be the first item, but better to sort in one place.